migrator.go

```diff
@@ -192,6 +233,39 @@ func (m Migrator) HasColumn(value interface{}, field string) bool {
 	return count > 0
 }
 
+func (m Migrator) MigrateColumn(value interface{}, field *schema.Field, columnType gorm.ColumnType) error {
+	if err := m.Migrator.MigrateColumn(value, field, columnType); err != nil {
+		return err
+	}
+	return m.RunWithValue(value, func(stmt *gorm.Statement) error {
+		var description string
+		m.DB.Raw(
+			"SELECT description FROM pg_catalog.pg_description WHERE objsubid = (?) AND objoid = (?)",
+			gorm.Expr(
+				"SELECT ordinal_position FROM information_schema.columns WHERE table_name = ? AND column_name = ?",
+				stmt.Table, field.DBName,
+			),
+			gorm.Expr(
+				"SELECT oid FROM pg_catalog.pg_class WHERE relname = ? AND relnamespace = "+
+					"(SELECT oid FROM pg_catalog.pg_namespace WHERE nspname = current_schema())",
```

Comment:
the `current_schema()` should use `m.CurrentSchema(stmt)`?

which is required if your table is defined in another namespace like `accounts.users`